include/zfeature_common.h

```diff
@@ -68,6 +68,7 @@ typedef enum spa_feature {
 	SPA_FEATURE_SPACEMAP_V2,
 	SPA_FEATURE_ALLOCATION_CLASSES,
 	SPA_FEATURE_RESILVER_DEFER,
+	SPA_FEATURE_ZSTD_COMPRESS,
```

Comment:
Shouldn't ZSTD be added to the end of this enum, to avoid renumbering the existing entries (breaking the ABI)

Reply:
Ehh... yes... it should...

Reply:
done, please check again and mark as resolved

Reply:
checked.
This can be marked as resolved by you @c0d3z3r0